app/bundles/EmailBundle/Translations/en_US/messages.ini

```
@@ -274,7 +274,15 @@ mautic.email.send.emailtype.transactional="Transactional"
 mautic.email.send.emailtype.marketing="Marketing"
 mautic.email.send.edit.email.desc="Edit email from here. New window will pop up and allows you to edit email"
 mautic.email.send.edit.email="Edit Email"
-mautic.email.send.example="Send Example"
+mautic.email.send.now="Send now "
+mautic.email.send.example="Send example"
+mautic.email.send.schedule="Schedule"
+mautic.email.send.schedule.update="Update schedule"
+mautic.email.send.schedule.cancel="Cancel schedule"
+mautic.email.send.continue="Continue sending email to contacts added to the segment after sending has started?"
+mautic.email.send.continue.tooltip="'No' will send the e-mail to whoever is in the segment at the time the e-mail is sent. 'Yes' will keep the e-mail published until the specified end date."
```

```suggestion
mautic.email.send.continue.tooltip="'No' will send the e-mail to whoever is in the segment at the time the email is sent. 'Yes' will keep the email published until the specified end date."
```
As we are using "email" in the rest of the file, let's not use "e-mail".